core/domain/activity_jobs_one_off.py

```diff
@@ -47,3 +55,34 @@ def map(item):
     @staticmethod
     def reduce(key, values):
         pass
+
+
+class SnapshotMetadataModelsIndexesJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Er, this is weird. Haven't I already reviewed this in #7874? I don't understand why it's showing up again here?